test/runtests.jl

```diff
@@ -60,9 +60,13 @@ include("spaces.jl")
 include("tensors.jl")
 include("diagonal.jl")
 include("planar.jl")
-if !(Sys.isapple()) # TODO: remove once we know why this is so slow on macOS
-    include("ad.jl")
+# TODO: remove once we know AD is slow on macOS CI
```

Comment:
This is a very strange construct. Why not
```julia
if !(Sys.isapple() && get(ENV, "CI", false))
    include("ad.jl")
end
```

Reply:
It's mostly taken from [here](https://github.com/Jutho/TensorKit.jl/blob/33f10bc8ee826540b9113654c96ebb34800d27df/test/tensors.jl#L58).
I think the biggest problem is that the environment variables are strings, so they really become `"true"` and `"false"`, which makes for an annoying interface.
I don't think the construction above works, because `"true"` would not evaluate as a bool...

Reply:
ok, `parse(Bool, get(ENV, "CI", "false"))`

Reply:
Sorry I missed this, I tried `get(ENV, "CI", "false") == "true"`. Does that work?